cloudbuild.yaml

```diff
@@ -1,78 +1,38 @@
-# Decrypt the file containing the key
 steps:
-  - id: 'decrypt'
-    waitFor: ['-']
-    name: 'gcr.io/cloud-builders/gcloud'
+  - id: 'store-market-secrets'
```

Comment:
We get the current ENV_VARS from the `SecretManager` and store them on the `.env.production` file on the docker image app's code so it's later used by the `create-react-app` webpack configurations to build the code with the correct client secrets